kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProduceAction.java

```diff
@@ -80,6 +83,8 @@ public ProduceAction(
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
   @Produces(MediaType.APPLICATION_JSON)
+  @PerformanceMetric("v3.records.create")
```

Comment:
I would call this `v3.records.produce-to-topic` instead. We will have `produce-to-partition` later.

Reply:
Fixed.
I also changed the resource names, so that they are a bit more hierarchical and allowing differentiation from the future v3 Consume and v3 Produce-to-partition APIs.

Reply:
And again, after shortening the identifiers by not trying to fit them in a RESTful paradigm, per your suggestion.